docs/semgrep-ci/network-broker.md

```diff
@@ -189,6 +191,17 @@ inbound:
     - url: https://git.example.com/*
       methods: [GET, POST]
 ```
+
+Semgrep also creates and updates [GitHub Checks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks#checks) when performing Managed Scans on pull requests. To ensure checks can be both created and updated, add the `PATCH` method to the preceding allowlist example, or add a separate entry to allowlist check updates:
+
+```
+inbound:
+  allowlist:
+    # allow PATCH requests to update checks
+    - url: https://git.example.com/api/v3/repos/:owner/:repo/check-runs/:id
+      methods: [GET, POST, PATCH]
```

Comment:
Can we just recommend including PATCH from the get-go? I'm worried people will see the first thing and stop reading.

Reply:
We definitely can; I went this way first because it's more "least-privilege" oriented, though. What's your thought on that? Not worth it?

Reply:
@chmccreery it looks like other folks have approved - are you open to me merging now, and we can consider the change if we find this version isn't helpful?